### ISSUE TITLE
ModelMakeCommand::handle() does not exist

### ISSUE THREAD

**Issue by iam-raihan** [flipboxstudio/lumen-generator#20]
i'm getting this error in console when trying to use any command like 
```
$ php artisan key:generate

[ReflectionException]
  Method Flipbox\LumenGenerator\Console\ModelMakeCommand::handle() does not exist
```

what can be the reason?
note: i'm using laravel/lumen-framework": "5.5

**Comment by emonacko** [comment]
same problem

**Comment by aqidd** [comment]
should be fixed after merging #17 
available on release `5.4.4`
